src/createStore.js

```diff
@@ -1,8 +1,11 @@
 import { createStore as originalCreateStore } from 'redux'
 
 export function createStore (reducer) {
-  const extension = window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
-  return originalCreateStore(reducer, extension)
+  let enhancer
+  if (process.env.NODE_ENV !== 'production' && window.__REDUX_DEVTOOLS_EXTENSION__) {
+    enhancer = window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
Yes. I have DevTools turned off when I measure it. I only enabled the DevTools before publishing to GitHub pages.